.travis.yml

```diff
@@ -4,9 +4,6 @@ sudo: required
 
 language: go
 
-# make this also work for forks
-go_import_path: github.com/kubernetes/kompose
```

Comment:
Why is this removed? It shouldn't have any effect on this, or does it?

Reply:
Yeah, I need to re-work this...